ui/src/ensembler/components/table/ListRouterVersionsForEnsemblerTable.js

```diff
@@ -50,16 +51,15 @@ export const ListRouterVersionsForEnsemblerTable = ({
 
   useEffect(() => {
     if (results.activeItems.length > 0){
-      setEnsemblerUsedByActiveRouterVersion(true)
-    } else {
-      if (currentRouterVersion.isLoaded && !currentRouterVersion.error) {
-        if (currentRouterVersion.data.length > 0){
-          setEnsemblerUsedByCurrentRouterVersion(true)
-          setEnsemblerUsedByActiveRouterVersion(false)
-        } else {
-          setEnsemblerUsedByCurrentRouterVersion(false)
-          setEnsemblerUsedByActiveRouterVersion(false)
-        }
+      setEnsemblerUsedByActiveRouterVersion(true);
+    }
+
+    if (currentRouterVersion.isLoaded && !currentRouterVersion.error) {
+      if (currentRouterVersion.data.length > 0){
+        setEnsemblerUsedByCurrentRouterVersion(true);
+        setFilteredActiveRouterVersions(results.activeItems.filter(item => currentRouterVersion.data.every((e => e.id !== item.id))));
+      } else {
+        setEnsemblerUsedByCurrentRouterVersion(false)
```

Comment:
These changes are made to allow the `ensemblerUsedByCurrentRouterVersion` and `ensemblerUsedByActiveRouterVersion` states to be set independently.